config.md

```diff
@@ -0,0 +1,2 @@
+* **height_or_width:** Constrain the "height", "width", or "both"
```

Comment:
Remove this file. I will push a proper README.

Reply:
I think many users perfer to have the .md file because then it adds the instructions directly in anki when they pull the config up, but a README would be fine as well

Reply:
Perfect. I did not know it.